index.html

```diff
@@ -39,15 +45,16 @@ <h1 class="CUserInterface">
 			<button id="addBtn" onclick="addToJSON()">Add post!</button>
 		</center>		
 
-		<div class="searchBar">
-			<button href="">newest</button>
-			<button href="#">featured</button>
-			<button href="#">frequent</button>
-			<button href="#">votes</button>
-			<button href="#">active</button>
-			<button href="#">unanswered</button>
-			<button href="#">Ask Question</button>
-		</div>
+		<!-- Represents the different tabs to filter the questions by subjects -->
+		<center class="searchBar">
+			<button>newest</button>
```

Comment:
Why was every `href` property removed?

Reply:
It was removed as it did not serve any purpose as of right now